### PR TITLE
Integrated Mayhem

### PR DESCRIPTION
**Pull request by bmwtsn098**

### CODE REVIEW

File: mayhem/fuzz.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package fuzz
```
Comment (xansec) [inline mayhem/fuzz.go]: Can we rename this to explain what's being fuzzed?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by bmwtsn098** [comment]
I made the requested changes. Let me know if everything looks good. Thanks

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**